src/regions/ca/info/ca-qc/clinics.fr.md

```diff
@@ -14,6 +16,20 @@ Si vous croyez avoir une infection au COVID-19, vous pouvez contacter, sans frai
 
 [Source: Governement du Québec](https://santemontreal.qc.ca/en/public/coronavirus-covid-19/#c35268)
 
+#### Cliniques pour enfants
```

Comment:
Should we say "Cliniques pour enfants & leurs familles"? It seems the parents can get tested too but i might be reading it wrong

Reply:
Ste Justine is usually just for peds though :/